code/modules/projectiles/guns/ballistic/pistol.dm

```diff
@@ -35,6 +35,17 @@
 /obj/item/gun/ballistic/automatic/pistol/m1911/no_mag
 	spawnwithmagazine = FALSE
 
+/obj/item/gun/ballistic/automatic/pistol/deltaelite
+	name = "\improper Colt Delta Elite"
+	desc = "A modification on the classic 1911 handgun, this gun is chambered in 10mm. Like it's predecessor, it suffers from low capacity."
```

Comment:
This description should read "...Like its predecessor..."

Reply:
Honestly the difference between its and it's is entirely pointless.
There's not even a reason why they shouldn't read the same.